Makefile

```diff
@@ -8,7 +8,7 @@ build_no_test:
 	SKIP_TESTS=true ./build.sh
 
 test-ui:
-	cd ui && npm install && npm test -- --watchAll=false
+	cd ui && npx eslint . && npm install && npm test -- --watchAll=false
```

Comment:
Does  the `npm install` need to happen before `eslint` here? Travis seems to be failing on missing packages.